www/includes/easyparliament/templates/html/mp/register.php

```diff
@@ -0,0 +1,94 @@
+<?php
+include_once INCLUDESPATH . "easyparliament/templates/html/mp/header.php";
+?>
+
+<div class="full-page">
+    <div class="full-page__row">
+        <div class="full-page__unit">
+            <?php include '_person_navigation.php'; ?>
+        </div>
+        <div class="person-panels">
+            <div class="sidebar__unit in-page-nav">
+                <div>
+                    <h3 class="browse-content"><?= gettext('Browse content') ?></h3>
+                    <ul>
+                      <li><a href="https://www.mysociety.org/2024/01/17/improving-the-register-of-mps-interests/">Download a spreadsheet</a></li>
+                      <?php include '_featured_content.php'; ?>
+                      <?php include '_donation.php'; ?>
+                </div>
+            </div>
+
+            <div class="primary-content__unit">
+
+              <?php if ($party == 'Sinn Féin' && in_array(HOUSE_TYPE_COMMONS, $houses)): ?>
+                <div class="panel">
+                    <p>Sinn F&eacute;in MPs do not take their seats in Parliament.</p>
+                </div>
+              <?php elseif (isset($is_new_mp) && $is_new_mp && count($recent_appearances['appearances']) == 0): ?>
+                <div class="panel panel--secondary">
+                    <h3><?= $full_name ?> is a recently elected MP &ndash; elected on <?= format_date($entry_date, LONGDATEFORMAT) ?></h3>
+
+                    <p>When <?= $full_name ?> starts to speak in debates and vote on bills, that information will appear on this page.</p>
+
+                  <?php if ($has_email_alerts) { ?>
+                    <a href="<?= WEBPATH ?>alert/?pid=<?= $person_id ?>#" onclick="trackLinkClick(this, 'alert_click', 'Search', 'Person'); return false;">Sign up for email alerts to be the first to know when that happens.</a>
+                  <?php } ?>
+                </div>
+              <?php endif; ?>
+
+                <?php if ($register_interests): ?>
+                <div class="panel register">
+                    <a name="register"></a>
+                    <h2>Register of Members&rsquo; Interests</h2>
+                    <p><b>New</b>: <a href="https://www.mysociety.org/2024/01/17/improving-the-register-of-mps-interests/">Download a spreadsheet of all Members Interests.</a></p>
+
+                    <p>
+                        <a href="<?= WEBPATH ?>regmem/?p=<?= $person_id ?>">View the history of this MP&rsquo;s entries in the Register</a>
+                    </p>
+                    <?php if ($register_interests['date']): ?>
+                        <p>Last updated: <?= $register_interests['date'] ?>.</p>
+                    <?php endif; ?>
+
+                    <?= $register_interests['data'] ?>
+
+
+                    <p>
+                         <a class="moreinfo-link" href="https://www.publications.parliament.uk/pa/cm/cmregmem/100927/introduction.htm">More about the register</a>
```

Comment:
Shall we update this to a more up-to-date page?

Reply:
Linked to main parliament register page - in general we'll want to come back and add better text.